src/components/Button.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = (props) => { // eslint-disable-line
+  return (
+    <>
+      <button type="button">{props.name}</button> {/*eslint-disable-line*/}
+    </>
+  );
+};
```

Comment:
- [ ] it accepts `a button name` as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png)). In line 7 just call {buttonName}